src/ImageSharp/IO/DoubleBufferedStreamReader.cs

```diff
@@ -13,10 +13,10 @@ namespace SixLabors.ImageSharp.IO
     /// A stream reader that add a secondary level buffer in addition to native stream buffered reading
     /// to reduce the overhead of small incremental reads.
     /// </summary>
-    internal class DoubleBufferedStreamReader : IDisposable
+    internal sealed class DoubleBufferedStreamReader : IDisposable
```

Comment:
Ah yes.... We should be sealing everything we can actually. There's definite advantages in the new compilers.

Reply:
Totally. There are some cases where it can avoid virtual calls when sealing the class. I think we've already got most of them.